src/inireader.cpp

```diff
@@ -100,41 +101,38 @@ int INIReader::ParseError() const
 	return _error;
 }
 
-std::string INIReader::Get(const std::string& section, const std::string& name, const std::string& default_value) const
+std::string_view INIReader::Get(std::string_view section, std::string_view name, std::string_view default_value) const
 {
 	std::string key = MakeKey(section, name);
 	// Use _values.find() here instead of _values.at() to support pre C++11 compilers
```

Comment:
Not our "bug", but we can safely migrate to `at()` here.

https://stackoverflow.com/a/70559609